src/api/app/controllers/staging/staged_requests_controller.rb

```diff
@@ -21,7 +21,7 @@ def create
       render_error(
         status: 400,
         errorcode: 'invalid_request',
-        message: "Could not assign requests #{unassigned_requests.to_sentence} to #{@staging_project}."
+        message: "Could not assign requests '#{unassigned_requests.to_sentence}' to #{@staging_project}."
```

Comment:
these are numbers, no? Wouldn't it make more sense to have them in [] then?

Reply:
Currently it would look like this:

```ruby
> [1,2,3].to_sentence
=> "1, 2, and 3"
```

Thinking about it... if people want to build tools around this it might be better to provide an output that is easier to parse.

Reply:
We could show the plain array here, but I guess to make it easy to parse we would need to provide each missing number as an XML element. @coolo Am I right?

Reply:
I don't think tools parse XML errors. If it's an error, it's for humans to understand IMO.

Reply:
And I mean "count not assign requests [#{[1,2,3].to_sentense}]"

Reply:
because "requests '1,2'" looks like the string 1,2 - not a list of numbers

Reply:
That human can also understand the error message "Could not assign requests 1, 2, and 3 to ..", no? :)

I for one would be very confused why it put the numbers in quotes - as it's normally done to mark a literal. So scratch the to_sentense and go with "Could not assign requests (1, 2, 3) to .." or "Could not assign requests () to .."

Reply:
You are ~~overengineering~~ overthinking this. This is just a tiny patch to make it clearer that the request number couldn't be parsed.

"Could not assign requests  to home:Admin:Staging:A." could mean anything.

"Could not assign requests '' to home:Admin:Staging:A." makes it clear that the request number couldn't be parsed / found. Be it because of a typo in the XML or whatnot.